src/main/blackbox/blackbox.c

```diff
@@ -1402,8 +1403,12 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_FEEDFORWARD_JITTER_FACTOR, "%d",  currentPidProfile->feedforward_jitter_factor);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_FEEDFORWARD_BOOST, "%d",          currentPidProfile->feedforward_boost);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_FEEDFORWARD_MAX_RATE_LIMIT, "%d", currentPidProfile->feedforward_max_rate_limit);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ANGLE_FEEDFORWARD, "%d",          currentPidProfile->pid[PID_LEVEL].F);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_LEVEL_GYRO_FF_SCALE, "%d",        currentPidProfile->levelGyroFeedforwardScale);
```

Comment:
```suggestion
        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_LEVEL_GYRO_FF_SCALE, "%d",        currentPidProfile->levelGyroFeedforwardScale);
        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_LEVEL_RESPONSE, "%d",             currentPidProfile->level_response);
```

Reply:
Why don't we like P term , D term and Feedforward here? level_response is much less readable than angle_feedforward. And you suggested angle_feedforward!

Reply:
`level_response` is the response time of the feedforward factor - the responsiveness, in the time frame, of the quad to the stick input.  it could be equally well named `level_responsiveness` or something else.  I don't care.  What happens is that higher values make the quad respond faster, slower values make the quad respond more slowly, to stick inputs.

It is neither a P, D or feedforward parameter.  It changes the time constant, or rise time of the feedforward element, as per the following image:

![LevelV2ResponseTest](https://user-images.githubusercontent.com/11737748/208153903-f37f2642-fb95-41cb-86b3-07482f27fc65.jpg)